sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/CallerInformationParametersShouldBeLastTest.cs

```diff
@@ -40,7 +40,8 @@ public void CallerInformationParametersShouldBeLast()
         [TestCategory("Rule")]
         public void CallerInformationParametersShouldBeLastInvalidSyntax()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\CallerInformationParametersShouldBeLastInvalidSyntax.cs",
+            // FIXME the update to .NET 5 changed this behavior from a raised issue to no issues raised
```

Comment:
I'm not sure if it's worth opening a ticket for this

Reply:
My concern is that we will never address this if we will not do it now; I would remove the comment.

Since the test is also checking that no errors are produced while analyzing invalid syntax I consider that it's still useful.